test/es-module/test-esm-no-extension.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const common = require('../common');
+const fixtures = require('../common/fixtures');
+const { spawn } = require('child_process');
+const assert = require('assert');
+
+const entry = fixtures.path('/es-modules/noext-esm');
+
+const child = spawn(process.execPath, [
```

Comment:
I might add a comment describing the purpose of this test (it's relatively clear from the filename, but might still be nice; I like the approach I've seen in a few sections of the codebase of:

```
// description of test
{
  // test content, in closure to keep it contained.
}
```

Reply:
fixed in 6a75b17398